test/integration/BTCDAIRebalancingManagerWrapper.spec.ts

```diff
@@ -0,0 +1,465 @@
+/*
+  Copyright 2018 Set Labs Inc.
```

Comment:
Let's organize the tests as well.

We should have a `wrapper/set_protocol` directory and a `wrapper/strategies` directory